src/App.tsx

```diff
@@ -18,19 +21,51 @@ function App() {
   const [user, setUser] = useState<User | null>(null);
 
   return (
-    <>
-      {/* pages to be updated once routing is provided */}
-      {/* <LoginContext.Provider value={{user, setUser}}>
-        <LoginPage />
-        <Footer />
-        <Menu/>
-      </LoginContext.Provider> */}
-      <div className="flex min-h-screen flex-col justify-between">
+    <LoginContext.Provider value={{ user, setUser }}>
+      <main className="flex min-h-screen flex-col justify-start">
         <Menu />
-        <Error404 />
+        {/* <LoginPage /> */}
+
+        <h1 className="section-heading">HAPPENING NOW</h1>
```

Comment:
Yeah I think this would make sense! I can do that in my code.

Reply:
Actually I think it might make sense to do this in another PR since it also requires some redirecting things, we can get this one merged in sooner, @melocule could we make a ticket for this?

Reply:
for now could we at least put all the homepage code in a Homepage component?

Reply:
done!